AndroidSDKCore/src/main/java/com/leanplum/messagetemplates/MessageTemplates.java

```diff
@@ -89,18 +89,18 @@
 
     // Open URL.
     static final String DEFAULT_URL = "http://www.example.com";
-
+    static final String DEFAULT_BASE_URL = "http://leanplum/";
     // Web interstitial values.
-    static final String DEFAULT_CLOSE_URL = "http://leanplum:close";
+    static final String DEFAULT_CLOSE_URL = DEFAULT_BASE_URL + "close";
     static final boolean DEFAULT_HAS_DISMISS_BUTTON = true;
 
     // HTML Template values.
     public static final String FILE_PREFIX = "__file__";
     public static final String HTML_TEMPLATE_PREFIX = "__file__Template";
-    static final String DEFAULT_OPEN_URL = "http://leanplum:loadFinished";
-    static final String DEFAULT_TRACK_URL = "http://leanplum:track";
-    static final String DEFAULT_ACTION_URL = "http://leanplum:runAction";
-    static final String DEFAULT_TRACK_ACTION_URL = "http://leanplum:runTrackedAction";
+    static final String DEFAULT_OPEN_URL = DEFAULT_BASE_URL + "loadFinished";
+    static final String DEFAULT_TRACK_URL = DEFAULT_BASE_URL + "track";
+    static final String DEFAULT_ACTION_URL = DEFAULT_BASE_URL + "runAction";
+    static final String DEFAULT_TRACK_ACTION_URL = DEFAULT_BASE_URL + "runTrackedAction";
 
   }
```

Comment:
Why are these changes a part of this commit?

Reply:
they were merged from PR to this branch. Then we will merge this one to develop.